src/NServiceBus.Core/Pipeline/WellKnownStep.cs

```diff
@@ -19,7 +19,6 @@ public class WellKnownStep
             this.stepId = stepId;
         }
 
-
```

Comment:
*second nitpick :wink:*

This line should be removed, but has nothing to do with the changes, since we're not touching this file after all. I think this file should be reverted completely.